### ISSUE TITLE
Fix display template for alert banner

### ISSUE THREAD

**Issue by andybroomfield** [localgovdrupal/localgov_alert_banner#27]
Now its a new entity we need to fix the template that is used to the alert banner style.

Examples (BHCC)
<img width="1048" alt="Screenshot 2020-06-22 at 6 27 33 pm" src="https://user-images.githubusercontent.com/1467480/85317333-22da4980-b4b6-11ea-8dae-86400315767b.png">
Minor

<img width="1049" alt="Screenshot 2020-06-22 at 6 27 21 pm" src="https://user-images.githubusercontent.com/1467480/85317328-21a91c80-b4b6-11ea-930a-2935286651ac.png">
Major

We should only offer a basic templating in the module, as themes will want to override it.
Make sure we supply a class name for the type of alert so themes can approitally colour them.

**Comment by andybroomfield** [comment]
Is missing a hook theme in the .module file.
Previously would have been
````php
/**
 * Implements hook_theme().
 */
function localgov_alert_banner_theme() {
  return [
    // Bundle our template with the module (which can be overridden in theme)
    'eck_entity__alert_banner' => [
      'base hook' => 'eck_entity',
    ],
  ];
}
````

**Comment by andybroomfield** [comment]
Before we do this, we need to make the entity fieldable and restore fields to it (blocked by #29)

**Comment by andybroomfield** [comment]
For reference, it is this [GDS pattern ](https://docs.publishing.service.gov.uk/manual/emergency-publishing.html#death-of-a-notable-person) that we are implementing.

In addition to what is on the GDS style guide, we have a hide button / link that should use JS to hide the banner (there is included JS in the ported module). This should *not* be displayed if the field `remove_hide_link` is checked on the edit form.

to support sites that do not show the alert banner title, there is now an additional field `display_title` like blocks, so the alert banner needs to account for this. Additionally a difference between the GDS pattern and BHCC is that, on minor alert and annoucements only, is that the title is inline.

**Comment by andybroomfield** [comment]
@danchamp Try to fix #38 in the same PR and let me know if that is possible.
